java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -46,10 +47,14 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
         baseTransaction.hget(key4, field1);
         baseTransaction.hdel(key4, new String[] {field1});
 
-        baseTransaction.sadd(key5, new String[] {"baz", "foo"});
-        baseTransaction.srem(key5, new String[] {"foo"});
-        baseTransaction.scard(key5);
-        baseTransaction.smembers(key5);
+        baseTransaction.rpush(key5, new String[] {value1, value2, value2});
+        baseTransaction.rpop(key5);
+        baseTransaction.rpopCount(key5, 2);
+
+        baseTransaction.sadd(key6, new String[] {"baz", "foo"});
```

Comment:
set commands can remain at key5 and list commands can be in key6, no need to modify